docker-compose.development.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.8"
+
+services:
+  todo-app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: dev
+    image: todo-app:dev
+    env_file:
+      - .env.development
+    ports:
+      - 5000:5000

```

Comment:
You are missing the bind mount that the exercise asked for. For development, we want to be able to modify files and see that reflected in the container immediately, without needing to rebuild or restart anything.